build.assets/Dockerfile-centos7-fips

```diff
@@ -79,6 +84,37 @@ ENV GOEXPERIMENT=boringcrypto \
     GOROOT="/opt/go" \
     PATH="/opt/llvm/bin:$PATH:/opt/go/bin:/go/bin:/go/src/github.com/gravitational/teleport/build"
 
+# Install Rust
+ARG RUST_VERSION
+ENV RUSTUP_HOME=/usr/local/rustup \
+     CARGO_HOME=/usr/local/cargo \
+     PATH=/usr/local/cargo/bin:$PATH \
+     RUST_VERSION=$RUST_VERSION
+
+RUN mkdir -p $RUSTUP_HOME && chmod a+w $RUSTUP_HOME && \
+    mkdir -p $CARGO_HOME/registry && chmod -R a+w $CARGO_HOME
+
+
+ARG LLVM_VERSION=7.0.1
```

Comment:
I would add a comment here that this specific version is FIPS certified so that we don't mistakenly upgrade it in the future.

Reply:
You're right. I made similar comment for a BORINGSSL_VERSION variable but it should also be mentioned here